src/spectral/utils/spectral-validator.js

```diff
@@ -57,12 +62,27 @@ const setup = async function(spectral) {
   spectral.registerFormat('oas3', isOpenApiv3);
 
   // load the spectral ruleset, either a user's or the default ruleset
-  const spectralRuleset = await config.getSpectralRuleset(
-    defaultSpectralRuleset
+  const spectralRulesetURI = await config.getSpectralRuleset(
+    defaultSpectralRulesetURI
   );
+  const spectralRuleset = await readRuleset(spectralRulesetURI);
 
+  // Combine user ruleset with the default ruleset
+  // The default ruleset will take precendence in any rules it defines (including extended rules)
+  const defaultRuleset = await readRuleset(defaultSpectralRulesetURI);
+  mergeRules(spectralRuleset.rules, defaultRuleset.rules);
```

Comment:
Might be worth trying to see if you can achieve the same merging by simply calling ` await spectral.loadRuleset([spectralRuleset, defaultSpectralRulesetURI]);`

Reply:
Yup this does achieve the same, I wasn't confident it did at first but it seems to be working as expect! I updated the PR with this